app/models/ncr/work_order.rb

```diff
@@ -171,8 +177,12 @@ def final_approver
       end
     end
 
-    def email_approvers
-      Dispatcher.on_proposal_update(self.proposal, self.modifier)
+    def budget_approvals
+      self.individual_approvals.offset(1)
```

Comment:
do we have any other way of identifying the flavor of approval it is, other than by its relative position? that feels brittle to me.